cmd/dptp-controller-manager/main.go

```diff
@@ -295,7 +300,7 @@ func main() {
 			syncPeriod := 24 * time.Hour
 			options.SyncPeriod = &syncPeriod
 		}
-		mgr, err := controllerruntime.NewManager(cfg, options)
+		mgr, err := controllerruntime.NewManager(&cfg, options)
```

Comment:
You can not pass a pointer to the loop var

Reply:
Sorry. One more thing that I need to remember.